<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editCourse</class>
 <widget class="QWidget" name="editCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 17, 60);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>Course Code:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_code">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);

color: rgb(0, 0, 0);
	border-style: solid;
	border-width: 0px;
	border-radius: 17px;

	padding: 8px;</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>Course Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);
color: rgb(0, 0, 0);

	border-style: solid;
	border-width: 0px;
	border-radius: 17px;

	padding: 8px;</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_days">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);

color: rgb(0, 0, 0);</string>
       </property>
       <item>
        <property name="text">
         <string>Sunday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tuesday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wednesday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thursday</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_time">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>Lecture Hall</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_hall">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <item>
        <property name="text">
         <string>3201</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1125</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3101</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3102</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3103</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">font: 13pt &quot;Optima&quot;;
background-color: #FF8C32;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-width: 1px;
	border-radius: 17px;
	border-color: #f0742f;
	padding: 8px;
</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
